tflint/client/decode_named_values.go

```diff
@@ -1,16 +1,19 @@
 package client
 
 import (
+	"fmt"
+
 	hcl "github.com/hashicorp/hcl/v2"
 	"github.com/terraform-linters/tflint-plugin-sdk/terraform/configs"
 	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/msgpack"
 )
 
 // Variable is an intermediate representation of configs.Variable.
 type Variable struct {
 	Name        string
 	Description string
-	Default     cty.Value
+	Default     []byte
 	Type        cty.Type
```

Comment:
Oh, I thought that `cty.Type` needs to be converted as well, but if you can't explicitly declare `cty.unknownType` in the Terraform config, it might be okay.

Reply:
Unfortunately it can =( With "any" keyword in type, so I will make another PR